### ISSUE TITLE
Database Idea

### ISSUE THREAD

**Issue by KalobTaulien** [EdmontonPy/edmontonpy#15]
This repo could come with a small .sqlite3 database file with a premade superuser (admin/admin or something) that also comes with any saved data, like historical meetups. I think that'd help keep everybody's data consistent. It should have the bare minimum, so it'd probably have:

* A superuser 
* A couple historical meetups 
* Other small dynamic data (if there is any)

I think this will help people get up and running faster, but also give them a solid way to change pieces of data and test if it breaks before making a PR. 

Thoughts?

**Comment by dgmouris** [comment]
I personally don't agree with checking in a database, but I feel like what we could do is extend the "Getting Started" guide which would include a management command to get started (that would populate some dummy data).

This could be a fantastic teaching point for exactly what you point out above. 
Teaching points:
1. Creating a superuser
2. Doing some basic migrations
3. Adding some basic dummy data (via management command or something else).

@KalobTaulien @data-get what do you folks think?

**Comment by KalobTaulien** [comment]
@dgmouris I see your point. Add a fixture then, with an auto loading management command. 

I don't think giving everybody a brand new blank database is a good way to go; it's effectively adding more work for people to get started. If it's a single management command to load a fixture with dummy data, that's cool imo. 

Here's what I currently see though. 
What EdmontonPy wants: More people to be involved (all levels included)
What EdmontonPy provides: A high barrier to entry for juniors

**Comment by dgmouris** [comment]
@KalobTaulien I agree with not giving people a blank brand new database.

I think that fixtures is a really good idea, so that people can have a little bit of data to poke around, and yes it would be a simple management command.

I agree with the high barrier for juniors, but this is something that I personally want to work on (and is something we can improve on). I'm going to be doing a talk about contributing to the repository in August once we have more tickets and more solidified process for PRs and such to go through:

1. Forking a repo for yourself
2. Creating the Code, with tests going over MagicMocks a bit.
3. Creating the Pull Request.
4. Hopefully getting it merged live.

**Comment by data-get** [comment]
A management command that populates the configured database would help to lower the barrier of entry. I would not want it to populate it with a lot of data since the maintenance of the data could be quite cumbersome, lets try to keep the models to a minimum. Putting those steps in the contributing guidelines file should be fine.

**Comment by KalobTaulien** [comment]
@data-get yeah that's fine. I mean, if you look at the site now there's virtually no data on it already. Just a couple meetups so people can work on a listing page at least would be super helpful.
